internal/cli/api/api.go

```diff
@@ -72,16 +75,43 @@ func convertAPIToCobraCommand(command api.Command) (*cobra.Command, error) {
 	shortDescription, longDescription := splitShortAndLongDescription(command.Description)
 
 	cmd := &cobra.Command{
-		Use:    commandName,
-		Short:  shortDescription,
-		Long:   longDescription,
-		Hidden: true, // TODO: part of CLOUDP-280743 Polish autogenerated docs
+		Use:   commandName,
+		Short: shortDescription,
+		Long:  longDescription,
 		RunE: func(cmd *cobra.Command, _ []string) error {
```

Comment:
NOTE: everything in `RunE` is temporary code. Just to give an idea what the code in CLOUDP-280745 will do. All code in here will be replaced